pet005/sub-01/ses-intervention/pet/sub-01_ses-intervention_task-eyes_pet.json

```diff
@@ -1,6 +1,7 @@
 {
 	"Manufacturer": "Siemens",
-	"TaskName": "Eyes open/closed",
+	"TaskName": "eyes",
```

Comment:
TaskName should match task label.

Reply:
Not sure why the validator did not catch this. @rwblair ?

Maybe means I should have a look into the validator to make sure it catches those.